build.assets/windows/build.ps1

```diff
@@ -351,18 +351,70 @@ function Build-Tsh {
         Write-Host "::group::Signing tsh..."
         Invoke-SignBinary -UnsignedBinaryPath "$UnsignedBinaryPath" -SignedBinaryPath "$SignedBinaryPath"
         Write-Host "::endgroup::"
+    }
+    Write-Host $("Built TSH in {0:g}" -f $CommandDuration)
+
+    return "$SignedBinaryPath"  # This is needed for building Connect and bundling the zip archive
+}
+
+function Build-Tctl {
```

Comment:
Can you abstract this + Build-Tsh logic into one function? I didn't do this when it was just connect and tsh because the process is very different, but it's nearly identical for tsh and tctl.